sample_building/sample_building.py

```diff
@@ -637,4 +637,4 @@ def calculate_first_rx_sample_time(first_pulse_num_samples_with_tr, txrate):
 
     first_rx_sample_index = int(first_pulse_num_samples_with_tr/2)
     first_rx_sample_time = float(first_rx_sample_index)/txrate
-    return first_rx_sample_time
```

Comment:
I think this needs to be a float, as used elsewhere in the code for finding sample times in floating-point seconds.
It is also a float in the protobuf for SigProcPacket